debian/jessie/foreman/rules

```diff
@@ -26,3 +26,9 @@ build:
 	/bin/sed -ri '1sXenv rubyXenv foreman-rubyX' bin/*
 	/bin/sed -ri 's~^BUNDLER_CMD=""~BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"~' script/foreman-rake
 	/bin/mkdir -p .ssh
+	# Ensure backwards compatiblity by having dynflowd aliased to foreman-tasks
+	/bin/mv /usr/sbin/dynflow-executor /usr/sbin/dynflowd
```

Comment:
I guess this should be handled in the rules file?

Reply:
It also breaks the build:

```
17:16:10 /bin/mv /usr/sbin/dynflow-executor /usr/sbin/dynflowd
17:16:10 /bin/mv: cannot stat '/usr/sbin/dynflow-executor': No such file or directory
```